CONTRIBUTING.md

```diff
@@ -27,9 +27,8 @@ the [puppet-dev mailing list](https://groups.google.com/forum/#!forum/puppet-dev
 
 ## Getting Started
 
-* Make sure you have a [Jira account](https://tickets.puppetlabs.com).
 * Make sure you have a [GitHub account](https://github.com/signup/free).
-* Submit a Jira ticket for your issue if one does not already exist.
+* Submit a GitHub Issue if one does not already exist.
```

Comment:
I'm wondering if we maybe we should drop this step entirely and just say if the pull request fixes an existing issue, then reference it in https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Reply:
I've updated this line to indicate that it's optional and updated line 88 to include the documentation you linked to. 

How does that all look to you?